ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/platform.py

```diff
@@ -95,6 +95,14 @@
             "non_mlag": 1020,
         },
     },
+    {
+        "platforms": ["7358X4"],
+        "management_interface": "Management1/1",
```

Comment:
Seems weird that 7368X4 has Ma0 but this one has Ma1/1. I trust you are right, but then we may need to fix the others. Can you please confirm?

Reply:
Yeah, I'm wondering about that - trying to hunt down a 7368 in the lab today to check. I have a 7358, hence the PR :)

Reply:
Turns out that Ma0 is _not_ optimal for these platforms. It is _not_ mapped to Ma1/2 and it is somehow dependent on the Strata agent to load, so it has a 5 min delay on startup.
We will stick with ma1/1. Thank you for listening :)